homeworks/homework-1/app/src/App.js

```diff
@@ -0,0 +1,29 @@
+import './App.css';
```

Comment:
All of the styles files would be better import below that all of dependencies.
Ideal structure for code files:

// 1 - dependencies from the npm
// 2 - dependencies from the another dirs
// 3 - dependencies from the current folder
// 4 - dependencies in the one level (styles etc.)
This increases the readability of the code, since there can be a lot of dependencies in the file, which is not uncommon in the React way.